src/io/pithos/xml.clj

```diff
@@ -362,6 +362,13 @@ Will produce an XML AST equivalent to:
                        "or did not validate against our published schema.")]
         [:RequestId reqid]
         [:HostId reqid]]
+       :invalid-cors-xml
+       [:Error
+        [:Code "MalformedCORSError"]
```

Comment:
according to http://docs.aws.amazon.com/AmazonS3/latest/API/ErrorResponses.html this should be `MalformedXML`